targets/targets.json

```diff
@@ -2000,6 +2000,15 @@
         "extra_labels_add": ["RBLAB_BLENANO"],
         "macros_add": ["TARGET_RBLAB_BLENANO"]
     },
+    "RBLAB_BLENANO2": {
+        "supported_form_factors": ["ARDUINO"],
+        "inherits": ["MCU_NRF52"],
+        "macros_add": ["BOARD_PCA10040", "NRF52_PAN_12", "NRF52_PAN_15", "NRF52_PAN_58", "NRF52_PAN_55", "NRF52_PAN_54", "NRF52_PAN_31", "NRF52_PAN_30", "NRF52_PAN_51", "NRF52_PAN_36", "NRF52_PAN_53", "S132", "CONFIG_GPIO_AS_PINRESET", "BLE_STACK_SUPPORT_REQD", "SWI_DISABLE0", "NRF52_PAN_20", "NRF52_PAN_64", "NRF52_PAN_62", "NRF52_PAN_63", "RBLAB_BLENANO2"],
```

Comment:
It does not look like you need the "RBLAB_BLENANO2" Macro.

Reply:
shouldn't it actually be in `extra_labels_add` then? that'd seem to match up with other targets now that i look again.

Reply:
It should probably be in neither. Adding it to `extra_labels_add` will do nothing, as the label `RBLAB_BLENANO2` is already there as it's the target.

Reply:
i see. It's gone now.